### DISCUSSION TITLE
Timeout bei Anfragen an id.ru.impfnachweis.info / 10.30.17.10

### DISCUSSION THREAD

**Discussion by ChristianFischerJena** [Digitaler-Impfnachweis/certification-apis#175] [Q&A]
Hallo zusammen,
können Sie mir kurz sagen, ob es zur Zeit ein Problem mit der Referenzumgebung gibt? Bei der Anfrage an https://id.ru.impfnachweis.info erhalten wir nur noch einen timeout.

Besten Dank!

**Comment by ManuelB** [comment]
Hab das gleiche Problem. Bin mir ziemlich sicher, dass das nicht an mir liegt, sondern an der Referenzumgebung.
